cmd/dagger/cmd/project/templates/hello.cue

```diff
@@ -0,0 +1,23 @@
+// The first time you run the hello
+// action as `dagger do hello --log-format plain`,
+// make sure to run `dagger project update` first,
+// so that all required dependencies are available.
+
+package hello
+
+import (
+	"dagger.io/dagger"
+	"dagger.io/dagger/core"
+)
+
+dagger.#Plan & {
```

Comment:
Since we want to incite people to use universe package and only `core` when it's necessary, I think it's better to use universe package in `hello`?

So you could update `core.#Pull` with `docker.#Pull` or `alpine.#Build` and `core.#Exec` with `bash.#Run` or `docker.#Run`.